src/Microsoft.CodeQuality.Analyzers/VisualBasic/QualityGuidelines/BasicMarkMembersAsStatic.Fixer.vb

```diff
@@ -13,5 +16,19 @@ Namespace Microsoft.CodeQuality.VisualBasic.Analyzers.QualityGuidelines
     Public NotInheritable Class BasicMarkMembersAsStaticFixer
         Inherits MarkMembersAsStaticFixer
 
+        Protected Overrides Function GetTypeArguments(node As SyntaxNode) As IEnumerable(Of SyntaxNode)
+            Return TryCast(node, GenericNameSyntax)?.TypeArgumentList.Arguments
+        End Function
+
+        Protected Overrides Function GetExpressionOfInvocation(invocation As SyntaxNode) As SyntaxNode
+            Return TryCast(invocation, InvocationExpressionSyntax)?.Expression
+        End Function
+
+        Protected Overrides Function GetSyntaxNodeToReplace(memberReference As IMemberReferenceOperation) As SyntaxNode
+            Dim syntax = MyBase.GetSyntaxNodeToReplace(memberReference)
+
+            ' VB operation tree seems to have an incorrect syntax node association.
```

Comment:
Is there a bug for this?

Reply:
We have made a call very early in IOperation development to not fix issues related to inconsistent syntax association with operations. I can file a bug, but it is likely not going to meet the bar, especially for this case where it is arguable whether `AddressOf M` should be deemed as leading to a method reference in VB, and not the child node `M` of the AddressOf expression.